sklearn/multioutput.py

```diff
@@ -539,6 +474,73 @@ def predict(self, X):
 
         return Y_pred
 
+
+class ClassifierChain(BaseChain):
```

Comment:
I did not understand which part is to be replaced with ClassifierMixin, suggest me.